test/models/test_approximate_gp.py

```diff
@@ -193,6 +216,8 @@ def test_inducing_point_init(self):
         model_2 = SingleTaskVariationalGP(train_X=train_X_2, inducing_points=5)
         model_2_inducing = model_2.model.variational_strategy.inducing_points
 
+        self.assertTrue(model_1_inducing.shape == (5, 1))
+        self.assertTrue(model_2_inducing.shape == (5, 1))
```

Comment:
can use equal here (and below)
```suggestion
        self.assertEqual(model_1_inducing.shape, (5, 1))
        self.assertEqual(model_2_inducing.shape, (5, 1))
```